src/smt/translate_exprs.rs

```diff
@@ -98,8 +98,13 @@ impl<'smt, 'ctx> TranslateExprs<'smt, 'ctx> {
         &mut self.fuel_context
     }
 
-    pub fn add_constant_exprs(&mut self, constant_vars: &[Ident], expr_to_analyse: &mut Expr) {
-        let mut collector = ConstantExprCollector::new(constant_vars);
+    pub fn add_constant_exprs(
```

Comment:
As discussed, I think the proper phrase would be "literal" instead of constant. And this function needs a comment explaining why it exists.